lldb/include/lldb/Target/Statistics.h

```diff
@@ -25,6 +26,7 @@ namespace lldb_private {
 
 using StatsClock = std::chrono::high_resolution_clock;
 using StatsTimepoint = std::chrono::time_point<StatsClock>;
+using Duration = std::chrono::duration<double>;
```

Comment:
Is this used anywhere?

Reply:
Originally I had this passed down to the invocation as well, with the intent of the duration being checked to see if a warn should be emitted to the user. 

Decided against that as this has already gotten somewhat complicated. I intend to use Duration in subsequent PR's so I left it moved out of StatsDuration, but if we want to move it back into it's original scope I'm fine with that.

Reply:
If it's unused in this PR, I'd rather remove it here. And add it when you have the need for it